_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -93,6 +93,8 @@ configure.args \
                     -DCMAKE_MODULE_PATH=${cmake_share_module_dir} \
                     -DCMAKE_FIND_FRAMEWORK=LAST \
                     -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+                    -DCMAKE_C_COMPILER=${configure.cc} \
+                    -DCMAKE_CXX_COMPILER=${configure.cxx} \
                     -Wno-dev
```

Comment:
There's no reason to think that this is more reliable than `default`. We use `default` all over base, without issue. Assembling `args` this way scatters the arguments about and is generally confusing.